apps/server/src/models/groups.ts

```diff
@@ -104,13 +99,16 @@ Groups.init(
 )
 
 Groups.belongsTo(Users, {
-  foreignKey: 'ownerId',
-  as: 'user',
+  foreignKey: 'adminId',
+  as: 'admin',
+  targetKey: 'id',
 })
 
-Groups.belongsTo(GroupWallet, {
-  foreignKey: 'id',
-  as: 'groupWallet',
-  targetKey: 'ownerId',
+Users.hasMany(Groups)
+
+GroupWallet.belongsTo(Groups)
```

Comment:
specify the foreignKey in this association as well

Reply:
Done. Thank you.